src/xpath.js

```diff
@@ -4,14 +4,17 @@ define([
     xpath
 ) {
     return {
-        makeXPathModels: function (form) {
+        // hashtagConfig has two properties:
+        //   hashtagDictionary: {hashtag expression: XPath expression}
+        //   hashtagTransformations: {hashtag prefix: function to return property}
+        makeXPathModels: function (hashtagConfig) {
```

Comment:
@orangejenny I changed it from `form` to `hashtagConfig`. the initialization of the parser still passes in the form as a form happens to conform to the interface